app/jobs/digitization/batch_metadata_ingestion_job.rb

```diff
@@ -53,7 +53,7 @@ def create_items_from_graph(graph, batch_ingest)
 
     graph.query(query_for_local_identifiers) do |statement|
       # Each of the discrete items will become its own object
-      book = batch_ingest.books.new(owner_id: batch_ingest.user_id)
+      book = Digitization::Book.new(owner_id: batch_ingest.user_id)
```

Comment:
Think we can just do something similar as before here and save an extra call to the DB below (`batch_ingest.books << book` does DB call) by just setting the batch_ingest id that we want?

```suggestion
      book = Digitization::Book.new(owner_id: batch_ingest.user_id, digitization_batch_metadata_ingest_id: batch_ingest.id)
```